newIDE/app/src/MainFrame/index.js

```diff
@@ -2199,10 +2207,29 @@ const MainFrame = (props: Props) => {
                   ).length,
                   onOpen: () => chooseProject(),
                   onCreate: () => openCreateDialog(),
+                  onCreateFromExampleShortHeader: onCreateFromExampleShortHeader,
+                  onOpenFromExampleShortHeader: async (
+                    storageProvider,
+                    fileMetadata
+                  ) => {
+                    await getStorageProviderOperations(storageProvider);
+                    const state = await openFromFileMetadata(fileMetadata);
+
+                    if (state) {
```

Comment:
you can probably only have one if:
`if (state && state.currentProject)`

Reply:
I copy-pasted without further thinking, good idea!

Reply:
When changing it, I realized that the formatting is prone to confusion: the second if is a oneliner.

Reply:
Good catch, let's keep the exact same logic as now to avoid a edge case or risk any change in the behavior